src/index.html

```diff
@@ -11,7 +11,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post">
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="name" autocomplete="off">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            name="age"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birthday">
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement">
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input type="email" placeholder="email@example.com" name="email">
+        </label>
+
+        <label>
+          Password:
+          <input type="password" name="password">
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="cat-lover" value="yes">
+          Yes
+          <input type="radio" name="cat-lover" value="no">
+          No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="bedtime">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input type="range" name="work-rating" value="0">
```

Comment:
if there are three or more attributes, put each on a new line
```suggestion
          <input
            type="range"
            name="work-rating"
            value="0"
          >
```